src/Sylius/Component/Shipping/Checker/ShippingMethodEligibilityChecker.php

```diff
@@ -13,16 +13,14 @@
 
 namespace Sylius\Component\Shipping\Checker;
 
-use const E_USER_DEPRECATED;
 use Sylius\Component\Shipping\Model\ShippingMethodInterface;
 use Sylius\Component\Shipping\Model\ShippingSubjectInterface;
+use const E_USER_DEPRECATED;
 
 @trigger_error(sprintf('The "%s" class is deprecated since Sylius 1.8, use "%s" instead.', 'Sylius\Component\Shipping\Checker\ShippingMethodEligibilityChecker', 'Sylius\Component\Shipping\Checker\Eligibility\CompositeShippingMethodEligibilityChecker'), E_USER_DEPRECATED);
```

Comment:
```suggestion

@trigger_error(sprintf('The "%s" class is deprecated since Sylius 1.8, use "%s" instead.', 'Sylius\Component\Shipping\Checker\ShippingMethodEligibilityChecker', 'Sylius\Component\Shipping\Checker\Eligibility\CompositeShippingMethodEligibilityChecker'), \E_USER_DEPRECATED);
```